airflow/dag_processing/manager.py

```diff
@@ -97,7 +97,6 @@ class DagFileStat:
 class DagParsingSignal(enum.Enum):
     """All signals sent to parser."""
 
-    AGENT_RUN_ONCE = "agent_run_once"
     TERMINATE_MANAGER = "terminate_manager"
     END_MANAGER = "end_manager"
```

Comment:
In a future PR these will be removed too, as there is a simpler way we can achieve this